app/routes/sites.js

```diff
@@ -1,7 +1,17 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
-	model(){
-		return this.store.findAll('company');
-	}
+  model(){
+    return this.store.findAll('company');
+  },
+
+  setupController(controller, model) {
+    this._super(controller, model);
+
+    controller.setProperties({
+      lat: 37.0625,
+      lng: -95.677068,
+      zoom: 4
+    });
+  }
```

Comment:
What is the purpose of this setupController? What would be an alternative solution that doesn't use setupController?